client/app/pages/queries-list/queries-list.html

```diff
@@ -51,6 +64,7 @@
           <a href="https://redash.io/help/user-guide/querying/writing-queries">query writing documentation</a>.
         </div>
 
+        <big-message ng-switch-when="archived" message="'Archive queries to list them here.'" icon="'fa-star'" />
```

Comment:
`icon`=`fa-star`?

Reply:
Also the message should be less of a CTA , maybe something like "Archived queries will be listed here.".